src/main/java/Duke.java

```diff
@@ -1,10 +1,116 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
 public class Duke {
+    public  Duke(){
+
+    }
+    ArrayList<Task> tasks = new ArrayList<>();
```

Comment:
```suggestion
    public Duke() {
    }

    ArrayList<Task> tasks = new ArrayList<>();
```
You have some extra spacing in your constructor function declaration